docker/package/tezos_baking_wizard.py

```diff
@@ -638,6 +646,27 @@ def import_baker_key(self):
                         )
 
                     else:
+                        connected_ledgers = list_connected_ledgers()
+                        # This step requires tezos-node to be running and bootstrapped in order to gather
+                        # the data about the ledger-stored addresses, so it's defined right before invoking
+                        # after the node was boostrapped.
+                        ledger_url_query = Step(
```

Comment:
I don't have a strong opinion on this, but I had a similar step with dependent options in the last version (see `get_key_mode_query`), and I followed a suggestion by @pasqu4le to make it into a function with the options argument. This allowed it to be near other steps where it belongs and also improved readability at a glance I guess. Could you change this to be in a similar fashion?

LGTM to me otherwise.